frontend/specs/spec-helpers.ts

```diff
@@ -65,13 +65,49 @@ export const jobFactory = Factory.Sync.makeFactory<Job>({
   worker_version: getCodeVersion(),
 });
 
+const uuidGetter = (count: number) => {
+  let uuid = crypto.randomUUID();
```

Comment:
:warning: (bikeshedding so feel free to ignore): This diff adds randomness into the tests... I've seen that used effectively for fuzz-testing but in general it seems like unit tests try to be more deterministic to reduce the number of independent variables.

But that might just be a me thing so I was curious. Also, this might be a fun discussion to have in our next [Roundtable](https://docs.google.com/document/d/1D1FNkjbCZxFNQLgh2FM_uiqn_7ifwi5r31P9MHs-FhI/edit) so I added it there

Reply:
I replaced all randomness within test factories with [sfc32](https://github.com/bryc/code/blob/master/jshash/PRNGs.md#sfc32).